python/test/t_PointToPointConnection_std.py

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+from openturns import *
+
+TESTPREAMBLE()
+
+try:
+    # Construction based on two functions
+    # Create an instance */
+    myFunc = PointToPointConnection(SymbolicFunction("x", "sin(x)"), SymbolicFunction("x", "cos(x)"))
+
+    print("myFunc=", myFunc)
+    # Get the input and output description */
+    print("myFunc input description=", myFunc.getInputDescription())
+    print("myFunc output description=", myFunc.getOutputDescription())
+    # Get the input and output dimension */
+    print("myFunc input dimension=", myFunc.getInputDimension())
+    print("myFunc output dimension=", myFunc.getOutputDimension())
+    # Connection on a point */
+    point = [1.0]*myFunc.getInputDimension()
+    print("point=", point)
+    print("myFunc(point)=", myFunc(point))
+    print("myFunc.gradient(point)=", myFunc.gradient(point))
+    print("myFunc.hessian(point)=", myFunc.hessian(point))
+    # Get the number of calls */
+    print("called ", myFunc.getCallsNumber(), " times")
+
+    # Construction based on a PointToFieldFunction followed by a FieldToPointFunction
+    # Create a KarhunenLoeveResult
+    mesh = IntervalMesher([9]).build(Interval(-1.0, 1.0)))
```

Comment:
extraneous parenthesis here, the script throws a syntax error

Reply:
@jschueller Not catched by any autobuilder?

Reply:
We just compare the diff, so we just have to make sure the expout is not empty